colibre/auto_plotter/cold_gas_relations.yml

```diff
@@ -350,4 +354,534 @@ H2_mass_star_formation_rate_100:
+    section: Cold Gas Fractions vs Surface Density
+
+sigma_hi_to_neutral_H_fraction_100:
+  type: "scatter"
+  legend_loc: "upper left"  
+  x:
+    quantity: "derived_quantities.mu_star_100_kpc"
+    units: "Solar_Mass / kpc**2"
+    start: 1e5
+    end: 1e10
+  y:
+    quantity: "derived_quantities.gas_HI_to_neutral_H_fraction_100_kpc"
+    units: "dimensionless"
+    start: 1e-1
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e5
+      units: "Solar_Mass / kpc**2"
+    end:
+      value: 1e10
+      units: "Solar_Mass / kpc**2"
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: Surface Density-HI Gas to Neutral Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is HI gas mass over neutral gas mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs Surface Density
+
+sigma_h2_to_neutral_H_fraction_100:
+  type: "scatter"
+  legend_loc: "upper left"  
+  x:
+    quantity: "derived_quantities.mu_star_100_kpc"
+    units: "Solar_Mass / kpc**2"
+    start: 1e5
+    end: 1e10
+  y:
+    quantity: "derived_quantities.gas_H2_to_neutral_H_fraction_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e5
+      units: "Solar_Mass / kpc**2"
+    end:
+      value: 1e10
+      units: "Solar_Mass / kpc**2"
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: Surface Density-H2 Gas to Neutral Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is H2 gas mass over neutral gas mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs Surface Density
+
+sigma_sf_to_sf_plus_stellar_fraction_100:
+  type: "scatter"
+  legend_loc: "upper left"  
+  x:
+    quantity: "derived_quantities.mu_star_100_kpc"
+    units: "Solar_Mass / kpc**2"
+    start: 1e5
+    end: 1e10
+  y:
+    quantity: "derived_quantities.gas_sf_to_sf_plus_stellar_fraction_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e5
+      units: "Solar_Mass / kpc**2"
+    end:
+      value: 1e10
+      units: "Solar_Mass / kpc**2"
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: Surface Density-SF Gas to SF Gas + Stellar Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is SF gas mass over SF gas mass + Stellar mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs Surface Density
```

Comment:
Again, section name should be consistent with the rest `Cold Gas Fractions ($\mu_*$)`